README.md

```diff
@@ -1,2 +1,89 @@
-# docker-cuda-tk1
-Docker development environment for the NVIDIA TK1
+# Docker-CUDA-Orin-Series
+Docker development environment for the NVIDIA Jetson Orin Series (Nano, NX, and AGX).
+
+This Docker development environment is designed for the NVIDIA Jetson Orin Series (Nano, NX, and AGX). It is purposed for developmental purposes, supporting a variety of projects for these devices, and providing custom-compiled support libraries such as JsonCpp and OpenBLAS. The environment utilizes CUDA version 11.8. 
```

Comment:
Erroneous comma before "and?" Does it utilize CUDA 11.8? I'm not sure what is meant by "supporting a variety of projects."